### PR TITLE
[LPS-148842 & LPS-148131] [CP 2.0] Fix Team Members Invitation Form With Raysource

### PR DESCRIPTION
**Pull request by iagochaves**
### What was done?
- Provided sessionId prop on team members table flow to ensure success on requesting raysource API
- Created function to filter roles that are currently available on raysource for each account
- Fix validation for remaining administrators available to invite on both Team Members page and Onboarding process

### CODE REVIEW

**Comment by iagochaves** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js [hunk 1]
```diff
@@ -46,10 +47,19 @@ const Pages = () => {
...
 	};
 
+	const getAvailableAdministratorAssets = () => {
+		if (project) {
+			return project.maxRequestors - totalAdministratorAccounts;
+		}
+	};
+
+	const availableAdministratorAssets = getAvailableAdministratorAssets();
+
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js:57]:
```suggestion
	const availableAdministratorAssets = project && project.maxRequestors - totalAdministratorAccounts;
```

**Review by jairmedeiros — APPROVED** [review]

**Review by igordmf — APPROVED** [review]
Good job!

**Comment by iagochaves** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**